From 13edc1da15275ffa64be6006478d4a1da1e0ccc9 Mon Sep 17 00:00:00 2001
From: Ke Qinghua <qinghua.ke@freescale.com>
Date: Thu, 24 Apr 2014 13:40:23 +0800
Subject: [PATCH 309/602] ENGR00309848 Failed to mount usb-disk Host port(when
 usb mouse cold boot). 100%

Fix u-disk can not recognize when usb mouse is plugged before u-disk is inserted

Signed-off-by: Ke Qinghua <qinghua.ke@freescale.com>
---
 evk_6sl/fstab.freescale           | 4 ++--
 sabreauto_6q/fstab.freescale      | 4 ++--
 sabreauto_6q/fstab_nand.freescale | 4 ++--
 sabresd_6dq/fstab.freescale       | 4 ++--
 sabresd_6sx/fstab.freescale       | 4 ++--
 5 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/evk_6sl/fstab.freescale b/evk_6sl/fstab.freescale
index 8e1a3c0..f1a8050 100644
--- a/evk_6sl/fstab.freescale
+++ b/evk_6sl/fstab.freescale
@@ -3,8 +3,8 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/devices/soc0/soc.2/2100000.aips-bus/2198000.usdhc/mmc_host/mmc1 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
-/devices/soc0/soc.2/2100000.aips-bus/2184200.usb/ci_hdrc.1/usb1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
+/devices/soc0/soc.2/2100000.aips-bus/2190000.usdhc/mmc_host /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.2/2100000.aips-bus/2184200.usb/ci_hdrc.1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/block/mmcblk0p5    /system      ext4    ro                                                                               wait
 /dev/block/mmcblk0p4    /data        ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
 /dev/block/mmcblk0p6    /cache       ext4    nosuid,nodev,nomblk_io_submit                                                    wait
diff --git a/sabreauto_6q/fstab.freescale b/sabreauto_6q/fstab.freescale
index dfd5b1d..f1d95ba 100644
--- a/sabreauto_6q/fstab.freescale
+++ b/sabreauto_6q/fstab.freescale
@@ -3,8 +3,8 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/devices/soc0/soc.1/2100000.aips-bus/2190000.usdhc/mmc_host/mmc0 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
-/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1/usb1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
+/devices/soc0/soc.1/2100000.aips-bus/2190000.usdhc/mmc_host /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1  /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/block/mmcblk0p5    /system      ext4    ro                                                                               wait
 /dev/block/mmcblk0p4    /data        ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
 /dev/block/mmcblk0p6    /cache       ext4    nosuid,nodev,nomblk_io_submit                                                    wait
diff --git a/sabreauto_6q/fstab_nand.freescale b/sabreauto_6q/fstab_nand.freescale
index 413b72d..1929817 100644
--- a/sabreauto_6q/fstab_nand.freescale
+++ b/sabreauto_6q/fstab_nand.freescale
@@ -5,8 +5,8 @@
 
 # below is for ubi support, uncomment to support ubifs(NAND)
 
-/devices/soc0/soc.1/2100000.aips-bus/2190000.usdhc/mmc_host/mmc0 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
-/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1/usb1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
+/devices/soc0/soc.1/2100000.aips-bus/2190000.usdhc/mmc_host /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/ubi0_0     /system		    ubifs 	ro                                      wait
 /dev/ubi0_3     /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
 /dev/ubi0_1     /cache          ubifs   nosuid,nodev                            wait
diff --git a/sabresd_6dq/fstab.freescale b/sabresd_6dq/fstab.freescale
index f282f9f..8d8bbeb 100644
--- a/sabresd_6dq/fstab.freescale
+++ b/sabresd_6dq/fstab.freescale
@@ -3,8 +3,8 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/devices/soc0/soc.1/2100000.aips-bus/2198000.usdhc/mmc_host/mmc1 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
-/devices/soc0/soc.1/2100000.aips-bus/2184000.usb/ci_hdrc.0/usb2  /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
+/devices/soc0/soc.1/2100000.aips-bus/2198000.usdhc/mmc_host /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.1/2100000.aips-bus/2184000.usb/ci_hdrc.0  /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/block/mmcblk0p5    /system      ext4    ro                                                                               wait
 /dev/block/mmcblk0p4    /data        ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
 /dev/block/mmcblk0p6    /cache       ext4    nosuid,nodev,nomblk_io_submit                                                    wait
diff --git a/sabresd_6sx/fstab.freescale b/sabresd_6sx/fstab.freescale
index f5f65eb..11e0272 100644
--- a/sabresd_6sx/fstab.freescale
+++ b/sabresd_6sx/fstab.freescale
@@ -3,8 +3,8 @@
 # The filesystem that contains the filesystem checker binary (typically /system) cannot
 # specify MF_CHECK, and must come before any filesystems that do specify MF_CHECK
 
-/devices/soc0/soc.1/2100000.aips-bus/2198000.usdhc/mmc_host/mmc1 /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
-/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1/usb1 /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
+/devices/soc0/soc.1/2100000.aips-bus/2198000.usdhc/mmc_host /mnt/media_rw/extsd vfat defaults voldmanaged=extsd:auto
+/devices/soc0/soc.1/2100000.aips-bus/2184200.usb/ci_hdrc.1  /mnt/media_rw/udisk vfat defaults voldmanaged=udisk:auto
 /dev/block/mmcblk0p5    /system      ext4    ro                                                                               wait
 /dev/block/mmcblk0p4    /data        ext4    nosuid,nodev,nodiratime,noatime,nomblk_io_submit,noauto_da_alloc,errors=panic    wait,encryptable=footer
 /dev/block/mmcblk0p6    /cache       ext4    nosuid,nodev,nomblk_io_submit                                                    wait
-- 
1.8.0

